blueocean-core-js/package.json

```diff
@@ -54,6 +53,7 @@
     "react-dom": "^15.1.0"
   },
   "devDependencies": {
+    "@jenkins-cd/design-language": "0.0.123-unpublished-jm-b",
```

Comment:
@sophistifunk not sure about this - shoudln't this be a published version? (same in all other package.json)